tests/expectations/metrics/test_core.py

```diff
@@ -125,7 +196,10 @@ def test_stdev_metric_pd():
         metrics_to_resolve=(desired_metric,), metrics=metrics
     )
     metrics.update(results)
-    assert results == {desired_metric.id: 1}
+    assert results == {desired_metric.id: result}
+
+
+# Add tests SQLAlchemy Spark for stdev and mean
```

Comment:
@TrangPham We should remove comments like this (or add them with an annotation with a date).  Thanks.